spec/steps/execute.fmf

```diff
@@ -48,6 +51,12 @@ description: |
     which case the first log will be considered as the main one
     (e.g. presented to the user for inspection).
 
+    The ``ARTIFACT`` is an optional section depending on the content of
+    test's :ref:`/spec/tests/artifact` attribute. Path to file or directory
+    is relative to the execute step working directory.
+
+    The ``DURATION`` is an optional section stating how many seconds did the test run.
```

Comment:
What about using `hh:mm:ss` format for the duration to make it more readable for longer tests? I like that the `results.yaml` file is readable for humans as well.

Reply:
no preference, makes sense